ethcore/sync/src/chain/mod.rs

```diff
@@ -676,6 +691,8 @@ pub struct ChainSync {
 	sync_start_time: Option<Instant>,
 	/// Transactions propagation statistics
 	transactions_stats: TransactionsStats,
+	/// Unfetched transactions
```

Comment:
Are these txs that we have seen announced but not fetched yet? Are they txs that we know we do not have or?